<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QSpinBox" name="spinBoxThreadhold">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
      <property name="value">
       <number>30</number>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="DropArea" name="labelDropArea">
      <property name="text">
       <string>Drop files here</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QComboBox" name="comboBoxMethod">
      <item>
       <property name="text">
        <string>DifferenceBlendTrail                  (полное вычитание, доступ через методы)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DifferenceBlendTrailV2              (полное вычитание, доступ через указательи)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DifferenceBlendTrailV3              (проверка различий и вычитание)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DifferenceBlendTrailV4              (проверка различий, серая шкала)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DifferenceBlendTrailV4Fast       (проверка различий, серая шкала, своя)</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>421</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionLoad"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad">
   <property name="text">
    <string>Загрузить</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DropArea</class>
   <extends>QLabel</extends>
   <header>features/droparea.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
